test/__snapshots__/warp-deprecated.spec.js.snap

```diff
@@ -2,75 +2,75 @@
+  "[91m[REMOVED][0m color-text-inverted -> see https://warp-ds.github.io/tech-docs/css/text-color to find suitable semantic replacement",
+  "[91m[REMOVED][0m color-text-inverted-subtle -> see https://warp-ds.github.io/tech-docs/css/text-color to find suitable semantic replacement",
+  "[91m[REMOVED][0m color-text-link -> see https://warp-ds.github.io/tech-docs/css/text-color to find suitable semantic replacement",
+  "[91m[REMOVED][0m color-text-link-hover -> see https://warp-ds.github.io/tech-docs/css/text-color to find suitable semantic replacement",
+  "[91m[REMOVED][0m color-text-link-visited -> see https://warp-ds.github.io/tech-docs/css/text-color to find suitable semantic replacement",
 ]
 `;
 
 exports[`deprecated warp classes > Emits a warning if semantic color classes used with -active-hover suffix are found 2`] = `
 [
-  "[93m[DEPRECATED][0m s-bg-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-bg-subtle-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-bg-primary-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-bg-negative-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-bg-positive-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-bg-warning-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-bg-info-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-bg-info-subtle-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-border-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-border-subtle-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-border-primary-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-border-negative-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-border-positive-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-border-warning-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-border-info-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-border-info-subtle-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-icon-active-hover -> use classes with '-selected-hover' if available'",
-  "[93m[DEPRECATED][0m s-icon-subtle-active-hover -> use classes with '-selected-hover' if available'",
+  "[91m[REMOVED][0m s-bg-active-hover -> use 's-bg-selected-hover' (if available)'",
```

Comment:
if we are so specific, do we still need the "if available" bit?

Reply:
The rule (`/^(s-.+)-active-hover$/`) is extremely forgiving. 😄  ...so there might not be a class with the suggested name.